manifests/check.pp

```diff
@@ -1,62 +1,25 @@
-# == Define consul::check
-#
-# Sets up a Consul healthcheck
-# http://www.consul.io/docs/agent/checks.html
-#
-# == Parameters
-#
-# [*ensure*]
-#   Define availability of check. Use 'absent' to remove existing checks.
-#   Defaults to 'present'
-#
-# [*http*]
-#   HTTP endpoint for the service healthcheck
-#
-# [*id*]
-#   The id for the check (defaults to $title)
-#
-# [*interval*]
-#   Value in seconds for the interval between runs of the check
-#
-# [*notes*]
-#   Human readable description of the check
-#
-# [*script*]
+# @summary Sets up a Consul healthcheck
+# @see http://www.consul.io/docs/agent/checks.html
+#
+# @param ensure Define availability of check. Use 'absent' to remove existing checks. Defaults to 'present'
```

Comment:
Do you not prefer the style where the description is on the line below?  (Also you can leave out `Defaults to 'present'` here as puppet-strings will produce that information itself).

Reply:
I removed the defaults mentions. My personal preference is to have it in a single line, and my guess is that we've a slight majority of modules with that style. But I can live with multiline as well.

Reply:
maybe that's something for our styleguide :thinking:

Reply:
My preference is the opposite :)

The upstream styleguide says use either, but presumably the author preferred the indented description style as that's used in all the examples.